graphql/resolvers/_mutations/publish.js

```diff
@@ -205,7 +206,11 @@ module.exports = async (
       campaignId = id
       await redis.setAsync(campaignKey, campaignId)
     }
-    const html = renderEmail(content, newsletterEmailSchema)
+
+    const emailSchema = repoName.startsWith('newsletter-editorial-')
```

Comment:
use `content.meta.template` instead. Repo prefix are primarily used for filtering repos from `orbiting` (only `newsletter-` currently—don't offer people to edit our code repos in publikator).

Reply:
content.meta.template is null in my test newsletter, probably needs some extra work on the frontend to store that on initial add.

Reply:
`documentPlain` takes care of that `document` doesn't